data/transactions/logic/debugger.go

```diff
@@ -16,294 +16,185 @@
 
 package logic
```

Comment:
Note: pretty much the entirety of the old `debugger(_test).go` has been moved to the new `debugger_legacy(_test).go`. Git does not do a great job of showing this

Reply:
If we go with calling this facility "EvalTracer" or similar, this code can go back to "debugger*.go"